slidez-fe-core/src/pages/dashboard/Dashboard.tsx

```diff
@@ -235,6 +266,7 @@ const Dashboard = () => {
                                         </div>
                                     </div>
                                 ))}
+                                {isNotEmptyPresentation()}
```

Comment:
Can you add styles to it?)
like 
HTML:
```<div class="no-found-presentaitons"><p>Some text here</p></div>```

CSS:
.no-found-presenation {
display:flex;
justify-content: center;
align-content: center;
width: 100%;
height: 100%; // Here can be some troubles
font-weight: 400;
font-size: 16px;
}